Add faq, intro, deploy, secure and reverse proxy for TiDB Dashboard (#2737)
pingcap/docs: dashboard/dashboard-intro.md

---
title: TiDB Dashboard Introduction
summary: Introduce TiDB Dashboard.
category: how-to
---

# TiDB Dashboard Introduction

> TiDB Dashboard is a graphical interface provided by TiDB since v4.0, which is used to monitor and diagnose TiDB clusters. TiDB Dashboard is built into the PD component of TiDB and does not require independent deployment.

Reviewer: ```suggestion
TiDB Dashboard is a Web UI for monitoring, diagnosing and managing the TiDB cluster, available since v4.0. It is built into the PD component and does not require an independent deployment.
```

![TiDB Dashboard interface](/media/dashboard/dashboard-intro.gif)

TiDB Dashboard is open-sourced on [Github](https://github.com/pingcap-incubator/tidb-dashboard).

This document introduces the main features of TiDB Dashboard. You can click links in the following sections to learn more details.

## Show the overall running status of TiDB cluster

> You can use TiDB Dashboard to learn the TiDB cluster's queries per second (QPS), execution duration, the SQL types that consume the most resources, and other overview information.

Reviewer 2: Reference: https://www.google.com/search?ei=ywbfXqGNOYzZhwOvqYT4Dw&q=execution+duration&oq=execution+duration&gs_lcp=CgZwc3ktYWIQAzIGCAAQFhAeMgYIABAWEB4yBggAEBYQHjIGCAAQFhAeMgYIABAWEB4yBggAEBYQHjIGCAAQFhAeMgYIABAWEB4yBggAEBYQHjIGCAAQFhAeOgQIABBHUOUFWOUFYJIIaABwAXgAgAHOAYgBzgGSAQMyLTGYAQCgAQKgAQGqAQdnd3Mtd2l6&sclient=psy-ab&ved=0ahUKEwjhntXr6fPpAhWM7GEKHa8UAf8Q4dUDCAw&uact=5

See [TiDB Dashboard Overview](/dashboard/dashboard-overview.md) for details.

## Show the running status of components and host

You can use TiDB Dashboard to view the running status of TiDB, TiKV, PD, TiFlash components in the entire cluster and the running status of the host on which these components are located.

See [TiDB Dashboard Cluster Information Page](/dashboard/dashboard-cluster-info.md) for details.

> ## Show distribution and trend of read and write traffic

Reviewer 2: ```suggestion
## Show distribution and trends of read and write traffic
```

The Key Visualizer feature of TiDB Dashboard visually shows the change of read and write traffic over time in the entire cluster in the form of heatmap. You can use this feature to timely discover changes of application modes or locate hotspot issues with uneven performance.

See [Key Visualizer Page](/dashboard/dashboard-key-visualizer.md) for details.

## Show a list of execution information of all SQL statements

> The execution information of all SQL statements is listed on the SQL Statements page. You can use this page to learn the execution duration and total execution times at all stages, which helps you analyze and locate the SQL queries that consume the most resource and improve the overall cluster performance.

Reviewer 2: ```suggestion
The execution information of all SQL statements is listed on the SQL Statements page. You can use this page to learn the execution time and total executions at all stages, which helps you analyze and locate the SQL queries that consume the most resources and improve the overall cluster performance.
```

See [SQL Statements Page of TiDB Dashboard](/dashboard/dashboard-statement-list.md) for details.

## Learn the detailed execution information of slow queries

> The Slow Queries page of TiDB Dashboard shows a list of all SQL statements that take a long time to execute, including the SQL texts and execution information. This page helps you locate the cause of slow SQL performance or performance jitter.

Reviewer 2: ```suggestion
The Slow Queries page of TiDB Dashboard shows a list of all SQL statements that take a long time to execute, including the SQL texts and execution information. This page helps you locate the cause of slow queries or performance jitter.
```

See [Slow Queries Page](/dashboard/dashboard-slow-query.md) for details.

## Diagnose common cluster problems and generate reports

> The diagnostic feature of TiDB Dashboard automatically determines whether some common risks (such as inconsistent configurations) or problems exist in the cluster, generates reports and gives operation suggestions, or compares the status of each cluster metric in different time ranges for you to analyze the possible problems.

Reviewer 2: ```suggestion
The diagnostic feature of TiDB Dashboard automatically determines whether some common risks (such as inconsistent configurations) or problems exist in the cluster, generates reports, and gives operation suggestions, or compares the status of each cluster metric in different time ranges for you to analyze possible problems.
```

See [TiDB Dashboard Cluster Diagnostics Page](/dashboard/dashboard-diagnostics-access.md) for details.

## Query logs of all components

> On the Search Logs page of TiDB Dashboard, you can quickly search logs of all running instances in the cluster by keywords, time range, and other conditions, and download these logs to your local.

Reviewer 2: ```suggestion
On the Search Logs page of TiDB Dashboard, you can quickly search logs of all running instances in the cluster by keywords, time range, and other conditions, package these logs, and download them to your local machine.
```

> See [Search Logs Page](/dashboard/dashboard-log-search.md) for details。

Reviewer 2: ```suggestion
See [Search Logs Page](/dashboard/dashboard-log-search.md) for details.
```

> ## Collect performance data of each component

Reviewer 2: ```suggestion
## Collect profiling data for each instance
```

> Advanced debugging feature: Profiles each component online, and analyze internal operations (executed during the analysis period) and proportions of component instances without a third-party tools.

Reviewer 2: ```suggestion
This is an advanced debugging feature that lets you profile each instance online and analyze various internal operations an instance performed during the profiling data collection period and the proportion of the operation execution time in this period without third-party tools.
```
